src/node_http2.cc

```diff
@@ -342,8 +342,8 @@ Http2Priority::Http2Priority(Environment* env,
                              Local<Value> weight,
                              Local<Value> exclusive) {
   Local<Context> context = env->context();
-  int32_t parent_ = parent->Int32Value(context).ToChecked();
-  int32_t weight_ = weight->Int32Value(context).ToChecked();
+  int32_t parent_ = parent->Int32Value(context).FromMaybe(0);
+  int32_t weight_ = weight->Int32Value(context).FromMaybe(0);
```

Comment:
We should not just continue in the case of a failed maybe, ignoring exceptions is worse than `.ToChecked()` imo.